Version Packages (#78)
team-plain/typescript-sdk: CHANGELOG.md

# @team-plain/typescript-sdk

## 2.17.0

### Minor Changes

> - d25f9eb: Update CreateThreadInput to take a `description` vs a `previewText`

Reviewer: ```suggestion
- d25f9eb: Update CreateThreadInput to take `description` instead of `previewText`
```
